### PR TITLE
ui_test: require colon after command

### PR DESCRIPTION
**Pull request by RalfJung**
Turns out we already use colons everywhere properly.

Also rename ui_test::comments to parser. I think that better describes its contents. :)

### CODE REVIEW

File: ui_test/src/parser.rs [hunk 1]
```diff
@@ -188,16 +184,19 @@ impl Comments {
                 self.error_pattern = Some((args.trim().to_string(), l));
             }
             "stderr-per-bitwidth" => {
+                // args are ignored (can be sue as comment)
```
Comment (oli-obk) [inline ui_test/src/parser.rs:187]:
```suggestion
                // args are ignored (can be used as comment)
```

**Comment by oli-obk** [comment]
Ah oops. I know where we don't tho: with `//~` style comments, after the `ERROR` part

**Comment by RalfJung** [comment]
Ah, I didn't even know we allow `:` there.
That would make the syntax without a level be `//~:`? That looks kind of odd...

**Comment by oli-obk** [comment]
Yea, maybe we should forbid `//~ ERROR: asdfg` and just require a space for consistency?

**Comment by RalfJung** [comment]
Or we have to require some kind of keyword for "match an arbitrary level please". That would be the more robust thing to do, it avoids a silent fallback.

**Comment by oli-obk** [comment]
At that point we can require the proper level name and add another `//@` command that can forcibly increase the threshold for levels that need annotations (instead of just picking the lowest one of any of the existing annotations like we do now).

**Comment by RalfJung** [comment]
Yeah that would work as well.

**Pull request merged**